packages/node_modules/@webex/plugin-memberships/src/memberships.js

```diff
@@ -321,6 +323,89 @@ const Memberships = SparkPlugin.extend({
       .then((res) => new Page(res, this.spark));
   },
 
+  /**
+   * Returns a list of memberships with details about the lastSeenId for each
+   * user, allowing a client to indicate "read status" in a space GUI
+   *
+   * This differs from the memberships.list() function in the following ways:
+   *  -- it accepts only a room or object with a valid roomId
+   *  -- no other options, eg: max, are considered
+   *  -- results are not paginated
+   *  -- memberships in the return do not include the
+   *     "created", "isRoomHidden", fields
+   *  -- memberships in the return do include the new
+   *    "lastSeenId", and "lastSeenDate" fields
+   *     these will not exist if the member has never "seen" the space
+   *
+   * In general this function should be used only when the
+   * client needs to access read status info.
+   *
+   * This function may be deprecated when this info is provided in the membership
+   * objects returned in the list function.
+   *
+   *
+   * @instance
+   * @memberof Memberships
+   * @param {Object} options
+   * @param {string} options.roomId
+   * @returns {Promise<MembershipObjectList>}
+   */
+  listWithReadStatus(options) {
+    const conversation = {
+      id: deconstructHydraId(options.roomId).id
```

Comment:
This is always assuming a room id is base64 encoded, yeah? On web, we don't have the public hydra ids. Not sure if we need a check for that or not?

Reply:
This is exactly why I wanted you to review!  I'll discuss with @mattnorris if there is an established pattern for dealing with this.

Reply:
As I look into this more deeply I see that the pattern of "assuming" hydra IDs is all over the "public" plugin module interfaces.   Will defer on solving this problem as part of this PR.

Reply:
Yes, for the public plugins we assume Hydra to reflect the Dev Portal.